shell.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+API_KEY="hardcoded_api_key"
```

Shellcheck

[warning] API_KEY appears unused. Verify use (or export if used externally).

(SC2034)